content/kubermatic/master/installation/add_seed_cluster/_index.en.md

```diff
@@ -0,0 +1,211 @@
+
+### 1. Install Kubernetes cluster
+
+First, you need to install a Kubernetes cluster with some additional components. After the installation of
+Kubernetes you will need a copy of the `kubeconfig` to create a configuration for the new Kubermatic
+master/seed setup.
+
+To aid in setting up the seed and master clusters, we provide [KubeOne](https://github.com/kubermatic/kubeone/),
+which can be used to set up a highly-available Kubernetes cluster. Refer to the [KubeOne readme](https://github.com/kubermatic/kubeone/)
+and [docs](https://github.com/kubermatic/kubeone/tree/master/docs) for details on how to use it.
+
+Please take note of the [recommended hardware and networking requirements](../../requirements/cluster_requirements/)
+before provisioning a cluster.
+
+### 2. Install Kubermatic Dependencies
+
+When using Helm 2, install Tiller into the seed cluster first:
+
+```bash
+kubectl create namespace kubermatic
+kubectl create serviceaccount -n kubermatic tiller
+kubectl create clusterrolebinding tiller-cluster-role --clusterrole=cluster-admin --serviceaccount=kubermatic:tiller
+helm --service-account tiller --tiller-namespace kubermatic init
+```
+
+#### NodePort Proxy
+
+Kubermatic requires the NodePort Proxy to be installed in each seed cluster. The proxy is shipped as a
+[Helm](https://helm.sh) chart in the kubermatic-installer repository.
+
+As the NodePort Proxy Docker image is in a private registry, you need to configure the Docker Pull Secret for
+the Helm chart. You can re-use the `values.yaml` used during the installation or create new one and configure it like
+so:
+
+```yaml
+kubermaticOperator:
+  # insert the Docker authentication JSON provided by Loodse here
+  imagePullSecret: |
+    {
+      "auths": {
+        "quay.io": {....}
+      }
+    }
+```
+
+#### Cluster Backups
+
+Kubermatic performs regular backups of user cluster by snapshotting the etcd of each cluster. By default these backups
+are stored locally inside the cluster, but they can be reconfigured to work with any S3-compatible storage.
+The in-cluster storage is provided by [Minio](https://min.io/) and the accompanying `minio` Helm chart.
+
+If your cluster has no default storage class, it's required to configure a class explicitely for Minio. You can check
+the cluster's storage classes via:
+
+```bash
+kubectl get storageclasses
+#NAME                 PROVISIONER            AGE
+#kubermatic-fast      kubernetes.io/gce-pd   195d
+#standard (default)   kubernetes.io/gce-pd   2y43d
+```
+
+{{% notice note %}}
+Minio does not use `kubermatic-fast` because it does not require SSD speeds. A larger HDD is preferred.
+{{% /notice %}}
+
+To configure the storage class and size, extend your `values.yaml` like so:
+
+```yaml
+minio:
+  storeSize: '200Gi'
+  storageClass: hdd
+```
+
+It's also advisable to install the `s3-exporter` Helm chart, as it provides basic metrics about user cluster backups.
+
```

Comment:
We should place an hint, that the seed cluster would need a backup itself and what we recommend. (Maybe separate issue)